plugin/src/sbt-test/paradox/snippets/src/main/paradox/group.md

```diff
@@ -0,0 +1,3 @@
+@@ snip [Group A](../../test/scala/Multiple.scala) { #multiple-something-else group=a }
+
+@@ snip [Group B](../../test/scala/Multiple.scala) { #parseint-imports group=b }
```

Comment:
This tests if the generated HTML is correct, but will the group=b snippet be hidden by default? I have tried it on a project and it looks like that both snippets are always visible.

Also if there are no tabbed snippets in the page, then a user will have no way of switching between the two. I think we should add a hover box or other interactive control that would allow switching between the alternatives.